### PR TITLE
Remove the RE Observe team and add the RE Autom8 team

### PR DESCRIPTION
**Pull request by issyl0**
± a few team members because I can't find the spreadsheet.

If anyone in @alphagov/re-autom8 doesn't like the idea of the Seal, it's
in a separate commit so I can easily back it out.

### CODE REVIEW

**Review by matthewcullum-gds — COMMENTED** [review]
Isnt the Autom8 team missing a few members? Mark hynes and I think Andy Paine might also need to be added.

File: config/alphagov.yml [hunk 1]
```diff
@@ -185,6 +185,25 @@ govuk-platform-health:
...
 
+
+re-autom8:
+  members:
+    - adityapahuja
+    - issyl0
+    - matthewcullum-gds
+    - philandstuff
+    - sakisv
+    - tlwr
```
Comment (tlwr) [inline config/alphagov.yml:200]:
```
mmhmm
bartlabno
stephengrier
```

are also autom8 team members

**Review by tlwr — APPROVED** [review]

File: config/alphagov.yml (hunk 1 above)
Comment (tlwr) [reply config/alphagov.yml:200]: i will use the new github suggestions feature

Comment (tlwr) [inline config/alphagov.yml:200]:
```suggestion
    - tlwr
    - mmhmm
    - bartlabo
    - stephengrier
    - andy-paine
```

**Comment by issyl0** [comment]
Oooh, that's nice.

**Review by tlwr — APPROVED** [review]

**Pull request merged**